src/main/java/mate.academy/NoComponentAnnotationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy;
+
+public class NoComponentAnnotationException extends RuntimeException {
+    public NoComponentAnnotationException(String mes) {
```

Comment:
Consider renaming the parameter `mes` to `message` for clarity. Additionally, ensure that the message provides enough context about the missing `@Component` annotation to help with debugging.